docs/storage.md

```diff
@@ -468,6 +469,81 @@ config:
   directory: ""
 ```
 
+### OCI
+
+[Oracle Cloud Infrastructure](https://www.oracle.com/cloud). To configure OCI Object Storage as Thanos Object Store, you need access to OCI Object Storage.
+You can get a free OCI Cloud Services by following the [Oracle Cloud Free Tier](https://www.oracle.com/cloud/free) page.
+More information about [Overview of Object Storage](https://docs.cloud.oracle.com/en-us/iaas/Content/Object/Concepts/objectstorageoverview.htm).
+
+OCI Object Storage supports three different providers, default, instance principle, and raw. 
```

Comment:
This should be instance _principal_.

Reply:
Thanks for the eagle eyes. Push the fix a minute ago.